adapters/handlers/grpc/v1/prepare_reply.go

```diff
@@ -550,6 +554,10 @@ func extractPropertiesAnswer(scheme schema.Schema, results map[string]interface{
 			continue
 		}
 		if prop.IsPrimitive {
+			class := getClass(className)
```

Comment:
**suggestion (non-blocking):**  what do you think if we  call it once before that section, because it's called with every `if block` ?

Reply:
I tried that, the problem is that there are section where the class can be nil so that means we can't check for `class == nil { return err }` at the top and would have to do it in the if anyway :) So I'd rather try to get the class just when necessary in the if cases.